README.md

```diff
@@ -250,6 +251,22 @@ Signature `CancellationToken -> CLIArguments seq -> string[] -> int`. Runs the t
 assembly and also overrides the passed `CLIArguments` with the command line
 parameters. All tests need to be marked with the `[<Tests>]` attribute.
 
+### `runTestsReturnLogs`
+
+Signature `CLIArguments seq -> string[] -> Test -> string`.
+Accepts the same arguments as `runTestsWithCLIArgs`, but returns the console output as a string.
+
+This is useful for interactive environments like [F# interactive](https://learn.microsoft.com/en-us/dotnet/fsharp/tools/fsharp-interactive/) and [Polyglot Notebooks](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode), where console output is not available but the returned string can be displayed instead.
+
```

Comment:
I plan to add a screen shot once I can show the proper package reference.
Something like `#r "nuget: Expecto, 10.2.0"`.